tests/migrations/test_operations.py

```diff
@@ -1632,6 +1632,50 @@ def test_rename_field(self):
         self.assertEqual(definition[1], [])
         self.assertEqual(definition[2], {'model_name': "Pony", 'old_name': "pink", 'new_name': "blue"})
 
+    def test_rename_field_with_db_column(self):
+        project_state = self.apply_operations('test_rfwdbc', ProjectState(), operations=[
+            migrations.CreateModel('Pony', fields=[
+                ('id', models.AutoField(primary_key=True)),
+                ('field', models.IntegerField(db_column='db_field')),
+            ]),
+        ])
+        operation = migrations.RenameField("Pony", "field", "renamed_field")
+        new_state = project_state.clone()
+        operation.state_forwards("test_rfwdbc", new_state)
+        self.assertIn('renamed_field', new_state.models['test_rfwdbc', 'pony'].fields)
+        self.assertNotIn('field', new_state.models['test_rfwdbc', 'pony'].fields)
+        self.assertColumnExists('test_rfwdbc_pony', 'db_field')
+        with connection.schema_editor() as editor:
+            with self.assertNumQueries(0):
+                operation.database_forwards('test_rfwdbc', editor, project_state, new_state)
+        self.assertColumnExists('test_rfwdbc_pony', 'db_field')
+        with connection.schema_editor() as editor:
+            with self.assertNumQueries(0):
+                operation.database_backwards('test_rfwdbc', editor, new_state, project_state)
+        self.assertColumnExists('test_rfwdbc_pony', 'db_field')
+
+    def test_rename_field_with_fk_db_column(self):
```

Comment:
Both tests are almost the same. I could do it with just one test and two columns.

I don't know the "Django best practice" here.

Reply:
We're just checking the number of queries, so a single test with two columns sounds fine.